### PR TITLE
Add sample CRD for LED light

### PR DESCRIPTION
**Pull request by sujithsimon22**
**What type of PR is this?**
 /kind feature

**What this PR does / why we need it**:
This PR provides sample CRDs for LED light. This provides an easy device model and device instance for the user to understand and will helpful for the users to create a device without making use of the defined protocols.

### CODE REVIEW

**Comment by sids-b** [comment]
I think this can be the last CRD sample in kubeedge repo. Any other samples we can add in examples repository.

File: build/crd-samples/devices/led-light-device-model.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+ properties:
+  - name: power-status
+    description: Indicates whether the led light is ON/OFF
+    type:
+      string:
+        accessMode: ReadWrite
+        defaultValue: 'OFF'
+  - name: gpio-pin-number
+    description: Indicates whether the GPIO pin to which LED is connected
```
Comment (rohitsardesai83) [inline build/crd-samples/devices/led-light-device-model.yaml]: Indicates ~whether~ the GPIO pin to which LED is connected

File: build/crd-samples/devices/led-light-device-model.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    type:
+      string:
+        accessMode: ReadWrite
+        defaultValue: 'OFF'
+  - name: gpio-pin-number
+    description: Indicates whether the GPIO pin to which LED is connected 
+    type:
+      int:
+        accessMode: Read
+        defaultValue: 18
```
Comment (rohitsardesai83) [inline build/crd-samples/devices/led-light-device-model.yaml]: any comments we can put what a default value of 18 indicates?

File: build/crd-samples/devices/led-light-device-model.yaml (hunk 1 above)
Comment (sujithsimon22) [reply build/crd-samples/devices/led-light-device-model.yaml]: done, thanks

File: build/crd-samples/devices/led-light-device-model.yaml (hunk 2 above)
Comment (sujithsimon22) [reply build/crd-samples/devices/led-light-device-model.yaml]: done, thanks

**Pull request merged**